luigi/rpc.py

```diff
@@ -104,22 +104,26 @@ def __init__(self, url='http://localhost:8082/', connect_timeout=None):
             connect_timeout = config.getfloat('core', 'rpc-connect-timeout', 10.0)
         self._connect_timeout = connect_timeout
 
+        self._rpc_retry_attemps = config.getint('core', 'rpc-retry-attempts', 3)
+        self._rpc_retry_wait = config.getint('core', 'rpc-retry-wait', 30)
+
         if HAS_REQUESTS:
             self._fetcher = RequestsFetcher(requests.Session())
         else:
             self._fetcher = URLLibFetcher()
 
     def _wait(self):
-        time.sleep(30)
+        logger.info("Wait for %d seconds" % self._rpc_retry_wait)
```

Comment:
Actually I meant as "doesn't flake8 complain?". I think you can give the `self._rpc_retry_wait` as a parameter to the logging framework, and letting it do interpolating as a part of it's display step.